src/mono/mono/mini/aot-compiler.c

```diff
@@ -6697,23 +6697,27 @@ emit_method_code (MonoAotCompile *acfg, MonoCompile *cfg)
 		 * - it allows the setting of breakpoints of aot-ed methods.
 		 */
 
-		// Comment out to force dedup to link these symbols and forbid compiling
+		// Enable to force dedup to link these symbols and forbid compiling
 		// in duplicated code. This is an "assert when linking if broken" trick.
-		/*if (mono_aot_can_dedup (method) && (acfg->aot_opts.dedup || acfg->aot_opts.dedup_include))*/
-			/*debug_sym = mono_aot_get_mangled_method_name (method);*/
-		/*else*/
+#if FALSE
```

Comment:
```suggestion
#if 0
```

Reply:
few other instances of this pattern are in these three files:
```
$ git grep -l "#if .*FALSE" :/*.c

src/mono/mono/metadata/class.c
src/mono/mono/mini/mini-generic-sharing.c
src/mono/mono/mini/simd-intrinsics.c
```